script/verification/SystemConfigUpgrade.s.sol

```diff
@@ -0,0 +1,82 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.15;
+
+import {console2 as console} from "forge-std/console2.sol";
+import {Vm} from "forge-std/Vm.sol";
+import {LibString} from "solady/utils/LibString.sol";
+import {SuperchainRegistry} from "script/verification/Verification.s.sol";
+import "@eth-optimism-bedrock/src/dispute/lib/Types.sol";
+import {ISystemConfig} from "./ISystemConfig.sol";
+import {IResourceMetering} from "./IResourceMetering.sol";
+import {MIPS} from "@eth-optimism-bedrock/src/cannon/MIPS.sol";
+import {Simulation} from "@base-contracts/script/universal/Simulation.sol";
+import {NestedMultisigBuilder} from "@base-contracts/script/universal/NestedMultisigBuilder.sol";
+
+contract SystemConfigUpgrade is SuperchainRegistry {
+    using LibString for string;
+
+    struct SysCfgVars {
+        address owner;
+        uint256 scalar;
+        bytes32 batcherHash;
+        uint256 gasLimit;
+        address unsafeBlockSigner;
+        IResourceMetering.ResourceConfig resourceConfig;
+        address batchInbox;
+        address gasPayingToken;
+        address l1CrossDomainMessenger;
+        address l1StandardBridge;
+        address l1ERC721Bridge;
+        address disputeGameFactory;
+        address optimismPortal;
+        address optimismMintableERC20Factory;
+    }
+
+    address public systemConfigAddress;
+    SysCfgVars previous;
+
+    constructor(string memory l1ChainName, string memory l2ChainName, string memory release)
+        SuperchainRegistry(l1ChainName, l2ChainName, release)
```

Comment:
Since `l1ChainName` and `l2ChainName` are storage variable names in the `SuperchainRegistry` contract, using them as argument names in the constructors shadows those variable names
```suggestion
    constructor(string memory _l1ChainName, string memory _l2ChainName, string memory _release)
        SuperchainRegistry(_l1ChainName, _l1ChainName, _release)
```

Reply:
[5098753](https://github.com/ethereum-optimism/superchain-ops/pull/418/commits/5098753ad37ac09680f6978ec39371d9a961e403)